pallets/ema-oracle/src/lib.rs

```diff
@@ -142,6 +145,13 @@ pub mod pallet {
 		/// Whitelist determining what oracles are tracked by the pallet.
 		type OracleWhitelist: Contains<(Source, AssetId, AssetId)>;
 
+		/// Asset id converter based on xcm locations
+		type CurrencyIdConvert: sp_runtime::traits::Convert<Location, Option<AssetId>>;
```

Comment:
i believe it is Location conversion.  Name suggest otherwise.

Reply:
yes, but we are naming it like this in runtime already :)

Reply:
i guess renaming would require bigger refactor, this is atleast consistent

Reply:
due to consistency reason, I will keep it as it is, only changing the comment.